js/src/edit-block/components/top-notice.js

```diff
@@ -81,11 +84,20 @@ const TopNotice = ( { editorMode, isOnboarding } ) => {
 					<div className="flex items-center">
 						<QuestionIcon />
 						<h4 className="text-lg font-semibold text-blue-900 ml-2">
-							{ __( 'Next step: Create your block template.', 'genesis-custom-blocks' ) }
+							{ __( 'Next step: Edit your block template.', 'genesis-custom-blocks' ) }
 						</h4>
 					</div>
 					<p className="text-sm mt-2 ml-2">
-						{ __( 'To display this block, you can use the Template Editor here or add this template file to your theme:', 'genesis-custom-blocks' ) }
+						{ __( 'Edit the template in the', 'genesis-custom-blocks' ) }
+						&nbsp;
+						<button
+							className="underline"
+							onClick={ () => setEditorMode( TEMPLATE_EDITOR_EDITING_MODE ) }
+						>
+							{ __( 'Template Editor', 'genesis-custom-blocks' ) }
```

Comment:
![in-sdfs](https://user-images.githubusercontent.com/4063887/118788963-e8aa3100-b859-11eb-952f-856b94c5189a.png)